src/cr/cube/matrix/measure.py

```diff
@@ -2310,10 +2339,11 @@ def _counts(self):
             counts = self._second_order_measures.column_comparable_counts.blocks
             # --- Get base values & *row* subtotals
             return [counts[0][0], counts[1][0]]
-        # --- Use *row* comparable counts (going across columns) ---
-        counts = self._second_order_measures.row_comparable_counts.blocks
-        # --- Get base values & *column* subtotals
-        return [counts[0][0], counts[0][1]]
+        else:
```

Comment:
i think this else is not needed

Reply:
Shoot, I must have started from a file that had already been edited, I didn't mean to change this line. Will investigate what happened

Reply:
Ok yea, must have started from an edited file, I did a rebase so I could make sure I only changed where I thought.